index.html

```diff
@@ -10,59 +10,77 @@
 </head>
 <body>
   <main id="app">
-    <div style="display: block">
-    <div class="div">
-      <img src="src/assets/git.png" width="100px" height="100px"/>
-    </div>
-    <div>
-    <form id="calendary_filter" style="margin-top: 15px" class="calendary_filter">
-        <select id="input_condition" name="condition" class="input">
-          <option value="after">
-            After of
-          </option>
-          <option value="before">
-            Before of
-          </option>
-          <option value="equals">
-            Equals to
-          </option>
-        </select>
-        <!-- <input 
-          type="text" 
-          placeholder="User/Organization" 
-          list="users_history"
-          id="input_user" 
-          name="user" 
-          class="input calendary_filter__repository_name" 
-          autofocus
-        />
-        <input 
-          type="text" 
-          placeholder="Repository name" 
-          list="repositories_history"
-          id="input_repository" 
-          name="repository" 
-          class="input calendary_filter__repository_name" 
-        /> -->
-        <div>
-          <button id="goback" class="btn--back">
-            <img src="./src/assets/arrow.png" style="margin-top: 3px" width="15px" height="15px" alt=""/>
-          </button>
-          <input type="date" name="date" id="input_date" class="calendary_filter__date_search" />
+    <div class="header">
+      <div class="logo">
+        <img src="./src/assets/logo.png" alt="">
+      </div>
+      <div class="language">
+        <div class="world">
+          <img src="./src/assets/world.png" alt="">
         </div>
-        <datalist id="repositories_history">
-          <!-- TODO: ADD repositories history on this datalist -->
-        </datalist>
-        <datalist id="users_history">
-          <!-- TODO: ADD users history on this datalist -->
-        </datalist>
+        <div class="content-select">
+          <select>
+            <option>English</option>
+            <option>Spanish</option>
+          </select>
+        </div>        
+      </div>
+    </div>
```

Comment:
In mobile the logo should be at the left bottom corner and the select at the right top corner, maybe to change the sizes in mobile to avoid that the page looks weird in mobile?

![imagen](https://user-images.githubusercontent.com/66505715/157699546-330fe0a9-1ac5-404e-b99e-3f502f939c5a.png)